pkg/tenant/config/config.go

```diff
@@ -34,6 +35,7 @@ type Config struct {
 	Auth         AuthConfig          `split_words:"true"`
 	Database     DatabaseConfig      `split_words:"true"`
 	Ensign       EnsignConfig        `split_words:"true"`
+	Topics       SDKConfig           `split_words:"true"`
```

Comment:
If Tenant needs more event streams in the future, my thinking was that we would just add more of these.